### PR TITLE
RD-2694 Remove jQuery usage from Manager.ts file.

### PR DESCRIPTION
**Pull request by stepek**
changed `$.param` to `stringify` from `queryString` lib and adjust tests.

### CODE REVIEW

**Comment by stepek** [comment]
## system tests 
https://jenkins.cloudify.co/view/UI/job/Stage-UI-System-Test/883/

File: app/utils/Manager.ts [hunk 1]
```diff
@@ -2,6 +2,8 @@
 /* eslint no-underscore-dangle: ["error", { "allow": ["_size", "_offset"] }] */
 
 import _ from 'lodash';
+import { stringify as stringifyQueryString } from 'query-string';
```
Comment (kubama) [inline app/utils/Manager.ts:5]: As for me the name `stringifyQueryString` implies a function that... stringifies a query string 🙂. How about naming it `stringifyToQueryString` or just `toQueryString`?

Comment (stepek) [reply app/utils/Manager.ts:5]: the usage of this import comes from previous commit changes, so IMO should be consistently

**Review by kubama — APPROVED** [review]

File: app/utils/Manager.ts (hunk 1 above)
Comment (stepek) [reply app/utils/Manager.ts:5]: maybe with ts migration can be done, Thanks!

**Pull request merged**